databuilder/task/neo4j_staleness_removal_task.py

```diff
@@ -55,22 +63,30 @@ def get_scope(self):
 
     def init(self, conf):
         # type: (ConfigTree) -> None
-        conf = Scoped.get_scoped_conf(conf, self.get_scope())\
-            .with_fallback(conf)\
+        conf = Scoped.get_scoped_conf(conf, self.get_scope()) \
+            .with_fallback(conf) \
             .with_fallback(DEFAULT_CONFIG)
         self.target_nodes = set(conf.get_list(TARGET_NODES))
         self.target_relations = set(conf.get_list(TARGET_RELATIONS))
         self.batch_size = conf.get_int(BATCH_SIZE)
+        self.dry_run = conf.get_bool(DRY_RUN)
         self.staleness_pct = conf.get_int(STALENESS_MAX_PCT)
         self.staleness_pct_dict = conf.get(STALENESS_PCT_MAX_DICT)
-        self.publish_tag = conf.get_string(JOB_PUBLISH_TAG)
+
+        self.ms_to_expire = None
+        if MS_TO_EXPIRE in conf:
```

Comment:
do we want to raise exception if both tag and MS_TO_EXPIRE are set?

Reply:
> do we want to raise exception if both tag and MS_TO_EXPIRE are set?

Good point. Will add.